relayer/processor/message_processor.go

```diff
@@ -347,6 +351,10 @@ func (mp *messageProcessor) trackAndSendMessages(
 
 		if broadcastBatch && (retries == 0 || ordered) {
 			batch = append(batch, t)
+			if len(batch) >= int(maxMsgNum) && maxMsgNum != 0 {
+				go mp.sendBatchMessages(ctx, src, dst, batch)
+				batch = nil
+			}
```

Comment:
just for clarity, is `batch` bound in size by the value of `maxMsgNum`? i.e. will `batch` always be less than or equal to `maxMsgNum`?

Reply:
yea, batch <= maxMsgNum